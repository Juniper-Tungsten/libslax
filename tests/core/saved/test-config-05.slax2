version 1.2;

main {
    copy-of configuration/protocols/bgp;
}
