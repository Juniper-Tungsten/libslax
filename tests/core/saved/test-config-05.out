<?xml version="1.0"?>
<bgp>
      <metric>15</metric>
    </bgp>
